### DISCUSSION TITLE
Car Diagnostics USB detection (M1 ARM)

### DISCUSSION THREAD

**Discussion by ncodee** [utmapp/UTM#5787] [macOS]
Anyone had any problems with car diagnostics tools such as VCDS with Windows on M1 Macs? 
I can't seem to get VCDS to detect the USB using a C-A connector using a Macbook.

Cheers

**Comment by echu2013** [comment]
I got usb passthrough working in the past with UTM but with ross-tech's devices you need to specify if using legacy cable (not supported on ARM because driver is unavailable) or using HEX-V2 and also which guest OS/Arch you are using. Generally for me I got both Windows x64 and ARM working and having USB passthrough functional, so the rest is upon drivers availability/functionality.

**Reply by ncodee** [reply]
Just saw your post now. I still can't get it to work, even with passthrough. Which version Windows are you using and did you have to turn off Memory Integrity off? The drivers don't install no matter what, and I can't get legacy drivers to install as it is greyed out for ARM. Thanks
